.github/dependabot.yml

```diff
@@ -1,13 +1,7 @@
 version: 2
 updates:
-- package-ecosystem: terraform
```

Comment:
I dont get why the /terraform update was removed

Reply:
We set this up a while ago, but in reality it hasn't done anything - we've never seen a PR from it. Arguably that could have been part of another work item.